client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/index.tsx

```diff
@@ -0,0 +1,169 @@
+							value="credentials"
+							name="how-to-access-site"
+							checked={ accessMethod === 'credentials' }
+							onChange={ handleAccessMethodChange }
+							disabled={ false }
+						/>
+					</div>
+					<div>
+						<FormRadio
+							label={ translate( 'Backup file' ) }
+							value="backup"
+							name="how-to-access-site"
+							checked={ accessMethod === 'backup' }
+							onChange={ handleAccessMethodChange }
+							disabled={ false }
+						/>
+					</div>
+				</div>
+				<hr />
+				{ accessMethod === 'credentials' && (
+					<div className="site-migration-credentials">
+						<div className="site-migration-credentials__form">
+							<div className="site-migration-credentials__form-field">
+								<FormLabel htmlFor="site-address">{ translate( 'Site address' ) }</FormLabel>
+								<FormTextInput
+									type="text"
+									id="site-address"
+									value={ siteAddress }
+									onChange={ ( e: ChangeEvent< HTMLInputElement > ) =>
+										setSiteAddress( e.target.value )
+									}
+								/>
+							</div>
+							<div className="site-migration-credentials__form-field">
+								<FormLabel htmlFor="username">{ translate( 'Username' ) }</FormLabel>
+								<FormTextInput
+									type="text"
+									id="username"
+									value={ username }
+									onChange={ ( e: ChangeEvent< HTMLInputElement > ) =>
+										setUsername( e.target.value )
+									}
+								/>
+							</div>
+							<div className="site-migration-credentials__form-field">
+								<FormLabel htmlFor="password">{ translate( 'Password' ) }</FormLabel>
+								<FormTextInput
+									type="password"
+									id="password"
+									value={ password }
+									onChange={ ( e: ChangeEvent< HTMLInputElement > ) =>
+										setPassword( e.target.value )
+									}
+								/>
+							</div>
+						</div>
+					</div>
+				) }
+				{ accessMethod === 'backup' && (
+					<div className="site-migration-credentials">
+						<div className="site-migration-credentials__form">
+							<div className="site-migration-credentials__form-field">
+								<FormLabel htmlFor="backup-file">{ translate( 'Backup file location' ) }</FormLabel>
+								<FormTextInput
+									type="text"
+									id="backup-file"
+									value={ backupFileLocation }
+									onChange={ ( e: ChangeEvent< HTMLInputElement > ) =>
+										setBackupFileLocation( e.target.value )
+									}
+									placeholder={ translate( 'Enter your backup file location' ) }
+								/>
+							</div>
+							<div>
+								{ translate(
+									'Share a link to your backup file, hosted on a service like Dropbox or Google Drive, ensuring that anyone with the link has access to the file.'
+								) }
+							</div>
+						</div>
+					</div>
+				) }
+				<div>
+					<NextButton type="submit">{ translate( 'Continue' ) }</NextButton>
+				</div>
+			</Card>
+		</form>
+	);
+};
+
+export type SiteMigrationIdentifyAction = 'continue' | 'skip_platform_identification';
```

Comment:
Looks good! Just a quick comment, Is this line needed here or just a residue?

Reply:
Thanks for catching it! It was indeed a residue!